kolibri/plugins/coach/assets/src/views/assignments/AssignmentSummary.vue

```diff
@@ -123,7 +123,7 @@
       changeStatus: 'Change',
       description: 'Description',
       noDescription: 'No description',
-      recipients: 'Recipients',
+      assignedGroupsListLabel: 'Visible to',
```

Comment:
:mood:  But now this does not match the modal...

Reply:
We should change the label on the modal then